java/android/nnstreamer/src/main/jni/nnstreamer-native-api.c

```diff
@@ -507,8 +510,8 @@ nns_parse_tensors_data (pipeline_info_s * pipe_info, JNIEnv * env,
 done:
   (*env)->DeleteLocalRef (env, data_arr);
 
-  if (failed) {
-    ml_tensors_data_destroy (*data_h);
+  if (failed && created) {
```

Comment:
How about removing the ```create``` variable and checking if ```data_h``` is null?

Reply:
If the param data_h is not null, we dont need releasing data handle.

Reply:
(*data_h) becomes non-null after 465.